backend/app/api/api_v1/endpoints/expectation.py

```diff
@@ -77,11 +81,10 @@ def list_expectations(
     )
 
     if include_history:
-        results = client.search(
-            body=validation.validations_query_body(datasource_id, dataset_id),
-            index=settings.VALIDATION_INDEX,
+        validations: list[Validation] = validation_repository.query_by_filter(
```

Comment:
The method is correctly typed, so you don't need to manually type-hint the result.